docs(readme): refactor of target-definition infos in readme (#5806)
eclipse-kura/kura: README.md

mvn -f kura/distrib/pom.xml help:all-profiles
```

> From the distrib folder, you can build only the Eclipse Kura Target Definition, useful for the development of addons, by running:

Reviewer: ```suggestion
Additionally you can build only the Eclipse Kura Target Definition, by running in the `distrib` folder:
```
